api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
This conditional name doesn't really make sense. Would probably be better to add a LambdaRuntime attribute to one of the region attribute mappings (this is how we determine EFS, etc)